doc/Makefile

```diff
@@ -171,7 +171,6 @@ MANPAGES = $(MANPAGES_7_GROFF) $(MANPAGES_5_GROFF) $(MANPAGES_3_GROFF) $(MANPAGE
 
 MANPAGES_BUILD = $(addprefix $(MANPAGES_BUILDDIR)/, $(notdir $(MANPAGES)))
 
-HTMLFILES = $(MANPAGES_BUILD:=.html)
```

Comment:
This seems to be used for man pages, not C++ documentation.

Reply:
Done.